### PR TITLE
chore: add price and with tooltips into table

### PR DESCRIPTION
**Pull request by dexturr**
Add price and size into orders table with tooltips

Screenshot 2023-09-07 at 15 29 56

### CODE REVIEW

File: frontend/components/receipts/utils/string-amounts/amount-with-tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            {amount}
+          </span>
+          <Info />
+        </span>
+      </Tooltip>
+      &nbsp;
+      <ExternalLink
+        data-testid={locators.assetExplorerLink}
+        className="text-vega-dark-300"
+        href={`${config.network.explorer}/assets/${assetId}`}
```
Comment (sam-keen) [inline frontend/components/receipts/utils/string-amounts/amount-with-tooltip.tsx:28]: Minor (non-blocking): this val is identical to 'assetHref' on line 13

**Review by sam-keen — APPROVED** [review]

**Comment by dexturr** [comment]
superceeded by #514

File: frontend/components/receipts/utils/order-table.spec.tsx [hunk 2]
```diff
@@ -9,10 +9,12 @@ describe('OrderTable', () => {
     // 1114-RCPT-013 I can see the direction of order I am submitting
     // 1114-RCPT-014 I can see the type of the order I am submitting
     // 1114-RCPT-015 I can see the reference of the order I am submitting
-    // 1114-RCPT-017 If present I can see the market id relating to the order
-    // 1114-RCPT-018 If present I can see the order id relating to the order
-    // 1114-RCPT-019 I can see the order id of the order I am amending
-    // 1114-RCPT-020 I can see the market id relating to the order I am amending
+    // 1114-RCPT-016 I can see the price of the order
```
Comment (dalebennett1992) [inline frontend/components/receipts/utils/order-table.spec.tsx:12]: 017

File: frontend/components/receipts/utils/order-table.spec.tsx [hunk 3]
```diff
@@ -9,10 +9,12 @@ describe('OrderTable', () => {
...
+    // 1114-RCPT-017 I can see the size of the order
```
Comment (dalebennett1992) [inline frontend/components/receipts/utils/order-table.spec.tsx:13]: 018

**Review by dalebennett1992 — CHANGES_REQUESTED** [review]
approbation numbering comments plus anymore I leave in the mean time lol

File: frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react'
+import { DecimalTooltip, locators } from './decimal-tooltip'
+import config from '!/config'
+
+describe('DecimalTooltip', () => {
+  it('renders and description the explorer link and docs links', () => {
+    // 1114-RCPT-024 I can see a tooltip for how to add the decimals to the number
+    // I can see a link in the tooltip to the relevant entity on the block explorer
```
Comment (dalebennett1992) [inline frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx:8]: missing AC link to 025

File: frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    const [tooltipAssetExplorerLink] = screen.getAllByTestId(locators.explorerLink)
+    expect(tooltipAssetExplorerLink).toBeInTheDocument()
+    expect(tooltipAssetExplorerLink).toHaveAttribute('href', 'https://example.com')
+    expect(tooltipAssetExplorerLink).toHaveTextContent('asset')
+
+    const [docsLink] = screen.getAllByTestId(locators.docsLink)
+
+    expect(docsLink).toBeInTheDocument()
+    expect(docsLink).toHaveAttribute('href', `${config.network.docs}/api/using-the-apis#decimal-precision`)
```
Comment (dalebennett1992) [inline frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx:33]: should there be AC for this link?

File: frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      'To get the value including decimals, find the asset on the block explorer and divide the amount by 10decimals e.g. 1000 with a decimals value of 2 would become 1000 ÷ 102 = 10.'
+    )
+
+    const [description3] = screen.getAllByTestId(locators.description3)
+    expect(description3).toBeInTheDocument()
+    expect(description3).toHaveTextContent('Read more')
+
+    const [tooltipAssetExplorerLink] = screen.getAllByTestId(locators.explorerLink)
+    expect(tooltipAssetExplorerLink).toBeInTheDocument()
+    expect(tooltipAssetExplorerLink).toHaveAttribute('href', 'https://example.com')
```
Comment (dalebennett1992) [inline frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx:27]: asset link too, should there be AC?

**Comment by dalebennett1992** [comment]
image
small bug here, right side of the text gets chopped. Can we fix?

**Comment by dalebennett1992** [comment]
image
second issue, links in tool tip do not open anything for me. If you hover or attempt to click you can see the url data in the bottom of the window, but it does not navigate or open anything new

**Comment by dalebennett1992** [comment]
image

the amount here looks like it is a link, but it is not. Can we remove the underline? Makes it harder to understand what is a link on the page and what isn't

File: frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx (hunk 6 above)
Comment (dexturr) [reply frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx:27]: This is general, takes any link so I think not.

File: frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx (hunk 5 above)
Comment (dexturr) [reply frontend/components/receipts/utils/string-amounts/decimal-tooltip.spec.tsx:33]: There is: `I can see a link in the tooltip to the relevant entity on the block explorer`

**Comment by dalebennett1992** [comment]
> image small bug here, right side of the text gets chopped. Can we fix?

image

Looks better!

**Pull request merged**